hw/top_englishbreakfast/data/top_englishbreakfast.hjson

```diff
@@ -649,23 +657,21 @@
   alert: [
   ]
 
-  // TODO: PINMUX
+  // TODO: need to overhaul this datastructure.
   pinmux: {
 
     // Total number of Multiplexed I/O
     //  All the input/outputs from IPs are muxed in pinmux, and it has # of I/O
-    //  talks to the outside of top_englishbreakfast.
+    //  talks to the outside of top_earlgrey.
     //  This field will be replaced to the length of PAD if padctrl is defined
-    num_mio: 32
+    num_mio: 44
 
     // Dedicated IO modules. The in/out ports of the modules below are connected
     //  to TOP IO port through PADS directly. It bypasses PINMUX multiplexers
     dio_modules: [
-      { name: "spi_device", pad: ["ChB[0..3]"] },
-      //{ name: "uart.tx", pad: ["ChA[0]"]},
-      { name: "uart", pad: ["ChA[0..1]"]},
```

Comment:
Note that when doing the rebase on top of #5255, you'll have to re-add `uart0` in the `mio` section.

Reply:
jep, done